javascript/README.md

```diff
@@ -0,0 +1,11 @@
+### Example of integrating Salt Edge API with JavaScript.
+
+* After clone this code, in folder with files run any file server (like python -m SimpleHTTPServer) or another.
+
+* In your [Saltedge Profile Settings](https://www.saltedge.com/clients/profile/edit) set this address to home url and save.
```

Comment:
Go to your Salt Edge profile settings, and set the url of the app to your external ip, so that CORS requests will be allowed by the Salt Edge API